modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaService.java

```diff
@@ -145,9 +151,7 @@ public <R> CompletableFuture<R> invoke(ClusterNode node, ReplicaRequest request)
      * @return A future holding the response.
      * @throws NodeStoppingException Is thrown when the node is stopping.
      */
-    public <R> CompletableFuture<R> invoke(ClusterNode node, ReplicaRequest request, String storageId)
-            throws NodeStoppingException {
+    public <R> CompletableFuture<R> invoke(ClusterNode node, ReplicaRequest request, String storageId) throws NodeStoppingException {
```

Comment:
It's definitely not your code, however let's also add  ReplicaUnavailableException into javadoc throws clause.

Reply:
Added